StarVMC/Geometry/StgmGeo/StgmGeo1.xml

```diff
@@ -1,968 +1,548 @@
+            <var name="xd"   value="PENT_hcdz * 1.72" comment="diameter of the connector part of the coax cable connector attached to edge bracket" />
+            <var name="xo"   value="PENT_hcdz * 3.0" comment="offset in x of the coax cable connector attached to edge bracket" />
+            <var name="xodz" value="PENT_hcdz * 2.8" comment="offset in z connector part of the coax cable connector attached to edge bracket" />
+        </Fill>
+        <Use struct="BRAC"  />
+
+        <Fill name="TFEE" comment="// frontEndElectronics"  >
+            <var name="Vers" value="1" comment="// version of FrontEndElectronics "  />
+            <var name="CardDX" value="1.47*0.110*INCH/2" comment="// 1.47 scale factor account for 9 lb of cables "  />
+            <var name="CardDY" value="PENT_ss * 0.70 / 2.0" comment="FEEL, replace with precise value from CAD mdoel"  />
+            <var name="CardDZ" value="TFEE_CardDY * 0.635135135 / 2.0" comment="FEES, replace with precise value from CAD model"  />
+            
+            <var name="PlateDX" value="0.110*INCH/2" comment="//... "  />
+            <var name="PlateDY" value="4.650*INCH/2" comment="//... "  />
+            <var name="PlateDZ" value="1.480*INCH" comment="//... "  />
+            
+            <var name="AssemblyThickness" value="TFEE_CardDX + TFEE_PlateDX" comment="//... "  />
+            <var name="RibDX" value="0.820*INCH/2 - 2*   TFEE_CardDX" comment="//... "  />
+            <var name="RibDY" value="TFEE_CardDX" comment="//... "  />
+            <var name="RibDZ" value="2.900*INCH/2 - 2*   TFEE_CardDX" comment="//... "  />
+            
+            <var name="Pos" value="{1.6,8,5}" comment="// Fee assembly brik size "  />
+        </Fill>
+        <Use struct="TFEE"  />
+
+
+        <Mixture name="G10" dens="1.7"  >
+            <Component name="Si" a="28.08" z="14" w="0.6*1*28./60."  />
+            <Component name="O" a="16" z="8" w="0.6*2*16./60. + 0.4*4*16./174."  />
+            <Component name="C" a="12" z="6" w="0.4*8*12./174."  />
+            <Component name="H" a="1"  z="1" w="0.4*14*1./174."  />
+        </Mixture>
+        
+        <Mixture name="HoneyComb" dens="0.282"  >
+            <Component name="Al" a="27" z="13" w="0.0105"  />
+            <Component name="N" a="14" z="7" w="0.7395"  />
+            <Component name="Adhesive" a="9" z="4.5" w="0.2500"  />
+        </Mixture>
+
+        <Mixture name="NPCO2" dens="0.2529E-02"  > <!--g/cm3-->
+            <Component name="C" a="12" z="6" w="28"  />
+            <Component name="H" a="1" z="1" w="54"  />
+            <Component name="O" a="16" z="8" w="11"  />
+        </Mixture>
+
+      <!-- Create and place the STGC Mother volume in the CAVE -->
+
+        <Create    block="STGM" />
+        <Placement block="STGM" in="CAVE" konly="ONLY" x="0" y="5.9" z="307.65" >
+            <Misalign table="Geometry/stgc/stgcOnTpc" row="0"/>
```

Comment:
The corresponding "ideal" misalignment tables should be a part of this PR.... 
StarDb/stgc/stgcOnTpc.20221220.000000.C   (explicit time stamp initialization)

Should return an ideal (unit matrix) table (eg StarDb/Geometry/tpc/TpcPosition.C)

Reply:
Addressed in 860d2af982f1094717c4fbbc522aee76fbbcfbdf